perf_test/sparse/tracked_testing.hpp

```diff
@@ -0,0 +1,27 @@
+//
+// Created by Poliakoff, David Zoeller on 4/26/21.
+//
+#ifndef KOKKOSKERNELS_TRACKED_TESTING_HPP
+#define KOKKOSKERNELS_TRACKED_TESTING_HPP
+#include <common/RAJAPerfSuite.hpp>
+#include <common/Executor.hpp>
+#include "KokkosSparse_spmv_test.hpp"
+
+
+
+
+
+namespace test {
+namespace sparse {
+void build_executor(rajaperf::Executor& exec, int argc, char* argv[], const rajaperf::RunParams& params) {
+  exec.registerGroup("Sparse");
+  for(auto* kernel : make_spmv_kernel_base(params)) {
+    exec.registerKernel("Sparse", kernel);
+  }
+  //for(auto* kernel : make_spgemm_kernel_base(argc, argv, params)) {
+  //  exec.registerKernel("Sparse", kernel);
+  //}
+}
+}
+}  // namespace test
```

Comment:
Is that tpls/rajaperf directory needed or a stray commit?

Reply:
@srajama1 -- I'm not seeing anything pertaining to tpls/rajaperf in this file ( perf_test/sparse/tracked_testing.hpp), so it might be a stray commit.  I'll look into it.